### PR TITLE
move pypi upload to a separate repo

### LINKED ISSUE #60: Use CircleCI deployment
instead of running at the end of the arch tests

### PR DESCRIPTION
**Pull request by Teyras**
Closes #60

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,6 +88,16 @@ jobs:
...
       - *doc
 
+  deploy:
+    docker:
+      - image: pritunl/archlinux:2017-07-08
+    working_directory: ~/cxflow
+    steps:
+      - *arch_deps
+      - checkout
+      - *install
```
Comment (FloopCZ) [inline .circleci/config.yml]: Is it really necessary to install it when building the wheel?

File: .circleci/config.yml [hunk 2]
```diff
@@ -105,3 +114,11 @@ workflows:
...
             - test_ubuntu_rolling
             - test_archlinux
+      - deploy:
+          filters:
+            branches:
+              only: master
+          requires:
+            - test_ubuntu_latest
+            - test_ubuntu_rolling
+            - test_archlinux
```
Comment (petrbel) [inline .circleci/config.yml]: I suggest adding `generate_documentation` as a deploy dependency as we probably want the up2date documentation

**Comment by petrbel** [comment]
@Teyras this PR closes #60, doesn't it?

File: .circleci/config.yml (hunk 2 above)
Comment (FloopCZ) [reply .circleci/config.yml]: You are right. We don't want to deploy the wheel when sphinx fails.

**Pull request merged**